### ISSUE TITLE
Reproduction of CIFAR-10 with 40 labels

### ISSUE THREAD

**Issue by YUE-FAN** [KyleZheng1997/simmatch#4]
Thanks for the great work! 

I have a question regarding the CIFAR experiments. The reported number is 94.40% on CIFAR10 with 40 labels, but our reproduction was only 75.25% after 547840 iterations. I guess there are important implementation details I missed from the paper. It would be very helpful if you could share your training scripts in this setting.

**Comment by KyleZheng1997** [comment]
Did you just simply run this code with CIFAR-10 40 labels without any changes? Our CIFAR experiments have very different settings.

**Comment by KyleZheng1997** [comment]
I have uploaded the code for CIFAR experiments [here](https://drive.google.com/file/d/1S59Eyt2klV02xW4FS46Mcir4L2jsIjeR/view?usp=sharing), please check it out and feel free to ask me if you have any further questions.

**Comment by BattashB** [comment]
Hi, I'm trying to recover the results on cifar100 with 400 labels, after 600 epochs I get 50% top1, I used the rar file attached in readme. Any idea why ?

**Comment by KyleZheng1997** [comment]
@BattashB Sorry I'm not quite sure why you got such low accuracy, but 50% Top-1 sounds like the performance of the online network. We actually report the performance of the ema network in our paper. BTW, we have provided the training logs, and you can check the Intermediate results to see where the gaps come from. (The training log for cifar100 with 400 labels should be named as c$seed$-4-full.log)

**Comment by KyleZheng1997** [comment]
@BattashB Well, It's really hard to tell. Would you like to share your code?  I will look at that when I'm free.

**Comment by JiHooooo** [comment]
i met the same problem with @BattashB, the accuracy of ema model is pretty low.  Have you soloved the problem?

**Comment by KyleZheng1997** [comment]
@JiHooooo Do you have any modifications on top of my shared code? If so, would you like to share your code? I could not help you guys without the code.

**Comment by JiHooooo** [comment]
@KyleZheng1997  Thanks for your quick reply. I have created a new repository to share my training code of cifar100. Could you help me to solve the problem? https://github.com/JiHooooo/SimMatch_cifar
